quinn-proto/src/config/mod.rs

```diff
@@ -159,6 +162,25 @@ impl EndpointConfig {
         self.rng_seed = seed;
         self
     }
+
+    /// Minimum size of the first Initial packet. You should NEVER consider calling this function
+    /// under any circumstances.
```

Comment:
The strength of this wording contradicts your intent in submitting this PR. If it's "NEVER" appropriate to call this, then we shouldn't merge the change. If the interface is worth having, then its documentation should explain when it makes sense to use.